source/assets/images/G-Logotype.svg

```diff
@@ -0,0 +1,10 @@
7.8447724,7.935156 L27.8447724,0.903636 C27.8447724,0.476316 28.1644607,0.146196 28.5802301,0.146196 L30.9375382,0.146196 C33.5439585,0.146196 35.3443451,1.990116 35.3443451,4.395276 L35.3443451,4.395276 Z M23.7754724,5.298516 L22.3185324,1.843236 L20.8608935,5.298516 L23.7754724,5.298516 Z M26.3228465,7.715556 C26.3822422,7.825356 26.405651,7.935156 26.405651,8.032716 C26.405651,8.435916 26.1093716,8.753076 25.7184086,8.753076 C25.3746126,8.753076 25.1377289,8.545356 25.0074078,8.227836 L24.3320445,6.628716 L20.3039721,6.628716 L19.6048505,8.276796 C19.4860592,8.582436 19.2379951,8.753076 18.9294871,8.753076 C18.5504032,8.753076 18.2548226,8.447796 18.2548226,8.057196 C18.2548226,7.947396 18.2897612,7.837956 18.3491568,7.715556 L21.4296941,0.635076 C21.5953031,0.256716 21.8912332,0.024876 22.3063039,0.024876 L22.3772292,0.024876 C22.7919505,0.024876 23.0767002,0.256716 23.2419598,0.635076 L26.3228465,7.715556 Z M15.0733164,1.513836 L12.9521934,1.513836 C11.8743373,1.513836 11.2342619,2.014596 11.2342619,2.930076 L11.2342619,2.954556 C11.2342619,3.820716 11.8980955,4.382676 12.941013,4.382676 L15.0733164,4.382676 L15.0733164,1.513836 Z M16.5299071,0.903636 L16.5299071,8.007876 C16.5299071,8.423676 16.2102188,8.753076 15.7961963,8.753076 C15.3930047,8.753076 15.0733164,8.423676 15.0733164,8.007876 L15.0733164,5.713596 L13.2369431,5.713596 L11.2223828,8.362116 C11.0571232,8.593956 10.8436482,8.753076 10.5120808,8.753076 C10.1564058,8.753076 9.81330862,8.472276 9.81330862,8.057196 C9.81330862,7.825356 9.90764287,7.666596 10.049843,7.483716 L11.6378029,5.420196 C10.52396,5.065956 9.75391298,4.235796 9.75391298,2.868876 L9.75391298,2.844396 C9.75391298,2.099916 10.0026759,1.464876 10.4411555,1.001196 C10.9743186,0.463716 11.7915327,0.146196 12.8341009,0.146196 L15.7961963,0.146196 C16.2102188,0.146196 16.5299071,0.476316 16.5299071,0.903636 L16.5299071,0.903636 Z M7.7603196,4.675716 L7.7603196,6.958476 C7.7603196,7.386156 7.59401182,7.691076 7.25021589,7.910676 C6.53956455,8.399196 5.54451292,8.838756 4.300349,8.838756 C1.70580778,8.838756 0.000104815832,6.933996 0.000104815832,4.443876 L0.000104815832,4.419036 C0.000104815832,2.026476 1.77673316,3.59999999e-05 4.27659075,3.59999999e-05 C5.52075467,3.59999999e-05 6.35019728,0.305676 7.10836513,0.866916 C7.25021589,0.976356 7.40464455,1.184436 7.40464455,1.476756 C7.40464455,1.880316 7.08460687,2.221956 6.6817647,2.221956 C6.49204804,2.221956 6.35019728,2.136276 6.23175539,2.051316 C5.69859219,1.648116 5.11791249,1.379196 4.21719511,1.379196 C2.68898028,1.379196 1.52797025,2.759076 1.52797025,4.395276 L1.52797025,4.419036 C1.52797025,6.177276 2.6533429,7.471476 4.34751613,7.471476 C5.130141,7.471476 5.84114173,7.214796 6.35019728,6.824196 L6.35019728,5.225436 L4.71507031,5.225436 C4.35974464,5.225436 4.06381461,4.944636 4.06381461,4.578156 C4.06381461,4.212036 4.35974464,3.918996 4.71507031,3.918996 L7.02521124,3.918996 C7.44028193,3.918996 7.7603196,4.248396 7.7603196,4.675716 L7.7603196,4.675716 Z"/>
```

Comment:
It would be cool if the logotype gradient could change with the mouse as well, but that might be a v2 ambition.